content/applications/sales/sales/products_prices/products/variants.rst

```diff
@@ -25,201 +25,147 @@ colors). Each one of these products has its own inventory, sales, etc.
 
-- Color: Red, Blue
+Attributes need to be created before product variants can be set up in your database. Attributes
+can be accessed via :menuselection:`Configuration --> Attributes`. To create a new Attribute,
+click **Create**. First, choose an attribute name, such as “color” or “size.”
 
-- Size: S, M, L, XL
+Then, select a display type from Radio, Select, and Color, which determines how your product will
+be shown on your eCommerce page, PoS dashboard or Product Configurator. Radio lists your options
+in a bullet style list; Select creates a dropdown menu; and Color creates little squares to
+choose the options from.
 
-In such a use case, you could create 1 template with three dimensions of
-variants (Layout, T-Shirts, Polos). But, it's recommended to create two
-different product templates as T-shirts may highly differ from polos or
-shirts and customer expect to see these as two different products in the
-e-Commerce:
+.. image:: variants/display-types-configurator.png
+   :align: center
+   :alt: Display Types on Product Configurator
 
-- Product Template: T-shirt
+Selecting a Variants Creation Mode informs Odoo when a new variant needs to be created once an
+attribute is added to a product. The available options are instantly, dynamically, or never.
+Instantly will create all possible variants as soon as an attribute and values are added to a
+product. Dynamically will create variants only when corresponding attributes and values are added
+to a sales order. Never as a selection means that variants are never created a particular attribute.
 
-  - Color: Red, Blue
-  - Size: S, M, L, XL
+.. note::
+   Once added to a product, an attribute’s Variants Creation Mode cannot be edited.
 
-- Product Template: Polos
+Values should be added to an attribute before saving, but more values can be added at any time,
+if needed. To add a value, click **Add a Line.** From there, you can:
 
-  - Color: Red, Blue
-  - Size: S, M, L, XL
+#. type in the value’s name
+#. check whether or not the value is custom
+#. specifically for colors, add an html color code to make it even easier for salespeople and
+   customers to know what they’re selecting.
 
-Configuration
-=============
+A color code can be selected by either dragging around the slider or entering a specific HTML
+color code to ensure an exact product match whenever possible.
 
-Activate the variant feature
-----------------------------
+.. image:: variants/picking-a-color.png
+   :align: center
+   :alt: Selecting a Color
 
-Before you can use product variants, you must first activate the product
-variants in the settings. To do so, you must go to the Sales app. In the
-menu :menuselection:`Configuration --> Settings`, locate the **Products Variants** line,
-and tick the option **Products can have several attributes**, then click
-on **Apply**.
+.. tip::
+   **Attributes** can also be created on the fly by adding a new line and typing the name into the
+   Variants Tab on a product.
```

Comment:
```suggestion
   *Attributes* can also be created on the fly by adding a new line and typing the name into the
   *Variants* tab on a product.
```
I prefer to use Italics when mentioning a field/feature name - having too much bold isn't great